exercises/protein-translation/src/lib.rs

```diff
@@ -1 +1,21 @@
+pub struct CodonsInfo;
 
+impl CodonsInfo {
+    pub fn name_for(&self, codon: &str) -> Result<&'static str, &'static str> {
```

Comment:
I'm not thrilled by returning `&'static str` in either position of the result; that's not normally great practice. While it works here, it wouldn't work if, for example, one were reading data from a file. 

I'm really tempted to try to get away with returning `&str` here and accepting it in `parse`, but that feels a bit mean. Probably the right thing to do here is make `CodonsInfo` generic over `'a`, and add a `std::marker::PhantomData<'a>` field to keep the compiler from complaining about unused generic types. Of course, that would also need a short comment explaining that the student should get rid of it once they have their own field using `'a`.